tests/e2e/performance/test_vdbench_workload.py

```diff
@@ -198,6 +190,44 @@ def test_vdbench_workload(
         sf_data = templating.load_yaml(template)
 
         target_results = template + 'Results'
+
+        log.info('Calculating Storage size....')
+        ceph_cluster = CephCluster()
+        total_capacity = ceph_cluster.get_ceph_capacity()
+        assert (total_capacity > constants.VDBENCH_MIN_CAPACITY), \
+            "Storage capacity is too low for performance testing"
+        log.info(f'The Total usable capacity is {total_capacity}')
```

Comment:
I would replace lines 199 with 197, so log will be presented before assertion

Reply:
i only want the log if not asserted.